Makefile.work

```diff
@@ -537,6 +537,7 @@ SONIC_BUILD_INSTRUCTION :=  $(MAKE) \
                            SONIC_INCLUDE_SYSTEM_GNMI=$(INCLUDE_SYSTEM_GNMI) \
                            SONIC_INCLUDE_SYSTEM_BMP=$(INCLUDE_SYSTEM_BMP) \
                            SONIC_INCLUDE_SYSTEM_EVENTD=$(INCLUDE_SYSTEM_EVENTD) \
+                           SONIC_SMARTSWITCH=$(SMARTSWITCH) \
```

Comment:
>SONIC_SMARTSWITCH [](http://example.com/codeflow?start=27&length=17)

Possible to get the info from other places? without adding an extra build-time config

Reply:
@ganglyu Can you explain why? I suppose if the problem is to have multiple values for different platforms. 

Then we should have the infra to for memory_checker to read from CONFIG_DB. As mentioned here: https://github.com/ganglyu/sonic-buildimage/blob/6270f971702bc1ae06df7da4aebcd280d40db0ff/files/image_config/monit/memory_checker#L270